exporter/exporterwrapper/exporterwrapper.go

```diff
@@ -29,41 +29,60 @@ import (
 	tracepb "github.com/census-instrumentation/opencensus-proto/gen-go/trace/v1"
 	"github.com/census-instrumentation/opencensus-service/data"
 	"github.com/census-instrumentation/opencensus-service/internal"
+	"github.com/census-instrumentation/opencensus-service/processor"
 	spandatatranslator "github.com/census-instrumentation/opencensus-service/translator/trace/spandata"
 )
 
-// OcProtoSpansToOCSpanDataInstrumented converts
-// OpenCensus Proto TraceData to OpenCensus-Go SpanData.
-// The "Instrumented" suffix serves to document that this
-// function is traced but also has stats for self-observability.
+// NewExporterWrapper returns a processor.TraceDataProcessor that converts OpenCensus Proto TraceData
+// to OpenCensus-Go SpanData and calls into the given trace.Exporter.
 //
 // This is a bootstrapping mechanism for us to re-use as many of
 // the OpenCensus-Go trace.SpanData exporters which were written
 // by various vendors and contributors. Eventually the goal is to
 // get those exporters converted to directly receive
 // OpenCensus Proto TraceData.
-func OcProtoSpansToOCSpanDataInstrumented(ctx context.Context, exporterName string, te trace.Exporter, td data.TraceData) (aerr error) {
+func NewExporterWrapper(exporterName string, ocExporter trace.Exporter) processor.TraceDataProcessor {
```

Comment:
Why there's only Tracing exporter wrapper? Also consider renaming this to `NewTraceExporterWrapper`.

Reply:
I had it as TraceExporterWrapper but decided to use ExporterWrapper for when we will wrap both.

Reply:
OK, make sense to me.